package.json

```diff
@@ -39,11 +43,11 @@
     "deepmerge": "^2.0.1",
     "ejs": "^2.6.1",
     "express": "^4.16.3",
-    "graphql": "^0.13.2",
+    "graphql": "^14.0.2",
```

Comment:
I assume this was just getting out of the "zero dot", and not 14 major revisions?

Reply:
Yes.